This patch comes from in-house; it will be submitted upstream once the
S12 -> 11.4 name change has been announced.

--- sendmail-8.15.2/include/sm/conf.h.orig	2015-06-17 09:51:58.000000000 +0000
+++ sendmail-8.15.2/include/sm/conf.h	2017-03-28 08:06:57.565907621 +0000
@@ -473,9 +473,9 @@
 #   ifndef HASGETUSERSHELL
 #    define HASGETUSERSHELL 0	/* getusershell(3) causes core dumps pre-2.7 */
 #   endif /* ! HASGETUSERSHELL */
-#   if SOLARIS < 21200
-#    define SIGWAIT_TAKES_1_ARG	1	/* S12 moves to UNIX V7 semantic */
-#   endif /* SOLARIS < 21200 */
+#   if SOLARIS < 21140
+#    define SIGWAIT_TAKES_1_ARG	1	/* S11.4 moves to UNIX V7 semantic */
+#   endif /* SOLARIS < 21140 */
 
 #  else /* SOLARIS */
 			/* SunOS 4.0.3 or 4.1.x */
